app/models/foreman_tasks/task/dynflow_task.rb

```diff
@@ -99,6 +99,32 @@ def running_steps
       execution_plan.try(:steps_in_state, :running, :suspended) || []
     end
 
+    def input_output_failed_steps
+      @failed_steps = failed_steps
+      @failed_steps.map do |f|
```

Comment:
Is an instance variable really needed for this?

Reply:
No, removed